ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/ProfileClustersTable.tsx

```diff
@@ -27,22 +29,37 @@ import StatusCountIcon from './components/StatusCountIcon';
 
 export type ProfileClustersTableProps = {
     currentDatetime: Date;
+    pagination: UseURLPaginationResult;
+    profileClustersResultsCount: number;
     profileName: string;
     tableState: TableUIState<ComplianceClusterOverallStats>;
 };
 
 function ProfileClustersTable({
     currentDatetime,
+    pagination,
+    profileClustersResultsCount,
     profileName,
     tableState,
 }: ProfileClustersTableProps) {
+    const { page, perPage, setPage, setPerPage } = pagination;
+
     return (
         <>
             <Toolbar>
                 <ToolbarContent>
                     <ToolbarItem>
                         <ProfilesTableToggleGroup activeToggle="clusters" />
                     </ToolbarItem>
+                    <ToolbarItem variant="pagination" align={{ default: 'alignRight' }}>
+                        <Pagination
+                            itemCount={profileClustersResultsCount}
+                            page={page}
+                            perPage={perPage}
+                            onSetPage={(_, newPage) => setPage(newPage)}
+                            onPerPageSelect={(_, newPerPage) => setPerPage(newPerPage)}
```

Comment:
Following up our discussion, `setPerPage` function also resets page to 1 since #7652
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/hooks/useURLPagination.ts#L26

Reply:
Oops, yeah I forgot to mention that to you after our last discussion on pagination